src/renate/updaters/avalanche/plugins.py

```diff
@@ -19,11 +19,6 @@ def _make_checkpoint_file_path(self, checkpoint_name: str) -> Path:
     def checkpoint_exists(self, checkpoint_name: str) -> bool:
         return self._make_checkpoint_file_path(checkpoint_name).exists()
 
-    def load_checkpoint(self, checkpoint_name: str, checkpoint_loader) -> Any:
-        checkpoint_file = self._make_checkpoint_file_path(checkpoint_name)
-        with open(checkpoint_file, "rb") as f:
-            return checkpoint_loader(f)
-
```

Comment:
This change has nothing to do with sphinx, is it intended?

Reply:
not sure why but sphinx will complain about that code. it is also identical to the parent's code, so we can drop it: https://github.com/ContinualAI/avalanche/blob/v0.3.1/avalanche/training/plugins/checkpoint.py#L335